### ISSUE TITLE
check-line-alignment not removing whitespace before start of  `@param` description

### ISSUE THREAD

**Issue by thernstig** [gajus/eslint-plugin-jsdoc#972]
## Expected behavior
I would have wanted the rule `"jsdoc/check-line-alignment": ["warn", "never"],` to remove unnecessary whitespace after the `-` but before the start of the description text. See ESLint sample below.

It is entirely likely that this is not a bug but a new feature request. Maybe an addition to `customSpacings`.

## Actual behavior
ESLint sample below shows it best.

## ESLint Config

```js
{
  "root": true,
  "reportUnusedDisableDirectives": true,
  "parserOptions": {
    "ecmaVersion": 2023,
    "sourceType": "module"
  },
  "env": {
    "node": true,
    "es2022": true
  },
  "extends": ["plugin:jsdoc/recommended"],
  "plugins": ["unicorn"],
  "rules": {
    "jsdoc/require-jsdoc": "off",
    "jsdoc/require-description": "warn",
    "jsdoc/check-syntax": "warn",
    "jsdoc/check-line-alignment": ["warn", "never"],
    "jsdoc/require-hyphen-before-param-description": [
      "warn",
      "always",
      { "tags": { "property": "always" } }
    ],
    "jsdoc/require-asterisk-prefix": "warn",
    "jsdoc/tag-lines": "off", // TODO: Temporary, will be fixed soon
    "jsdoc/no-bad-blocks": ["warn", { "preventAllMultiAsteriskBlocks": true }]
  },
  "settings": {
    "jsdoc": {
      "mode": "typescript"
    }
  }
}
```

### Before linting

```js
/**
 * Fixme
 *
 *   @param {Array}   requests - Fixme
 * @param {string} action   -   Fixme
 *
 * @returns {Array<any>} result -   Fixme
 */
function streamRequest(requests, action) {
  return [requests, action];
}
```

### After linting with `--fix`

Notice the white space here `-   Fixme` which I would have thought should become `- Fixme`

```js
/**
 * Fixme
 *
 * @param {Array} requests - Fixme
 * @param {string} action -   Fixme
 *
 * @returns {Array<any>} result -   Fixme
 */
function streamRequest(requests, action) {
  return [requests, action];
}
```

## Environment

- Node version: v18.12.1
- ESLint version v8.31.0
- `eslint-plugin-jsdoc` version: 39.6.4

**Comment by thernstig** [comment]
@brettz9 I believe the logic in the PR needs to be amended a bit.

Something like this:

```js
   * @param {string|string[]|TemplateResult|TemplateResult[]} event.detail.description -
   *    Notification description
```

Gets changed into this:

```js
   * @param {string|string[]|TemplateResult|TemplateResult[]} event.detail.description - 
   *    Notification description
```

It is subtle, but if you check right after the `-` it contains a space now.

My opinion is that if there is a line break directly after the `-` it should not add a space, as most other formatters (Prettier) will remove it, and then you get into a loop.
